src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -625,12 +625,12 @@ export class MatAutocompleteTrigger
         {injector: this._environmentInjector},
       );
     });
-    const optionChanges = this.autocomplete.options.changes.pipe(
```

Comment:
Can you add a test case for this?

Reply:
I added a test and ran the presubmit internally. I'm guessing I need the resulting cl to be merged before I can submit this?

Reply:
Yeah, lmk when you submit the CL and I'll merge this for you

Reply:
I've gone ahead and submitted the cl